internal/pkg/cli/flag.go

```diff
@@ -33,13 +33,19 @@ const (
 	manifestFlag       = "manifest"
 	resourceTagsFlag   = "resource-tags"
 
+	// Deploy flags.
+	yesInitWorkloadFlag = "init-wkld"
+	noInitWorkloadFlag  = "no-init-wkld"
```

Comment:
Just a thought. Should this be a bool instead? I understand wanting consistency with the other flags (deploy/no-deploy). 🤷🏼

Reply:
You don't actually get the right amount of granularity if it's just a bool. 

For example, if we just use `init-wkld` and the flag is un-specified, it's false. Then do you prompt for initialization, or do you assume the user's intent is not to init the workload and error out? It leads to problems where there's no way to force the command to exit when the intent is not to init the workload if it doesn't exist, or there's no way to display a prompt when the intent is ambiguous. Perhaps a pointer to a bool would do it, but I don't think there's a way of specifying a bool flag like `--init=false` in cobra. 

Long winded way of saying that as far as I can tell, the best way to specify choices like this where we want to prompt by default but allow customers to specify either a positive or a negative is to use two mutually exclusive bool flags.

Reply:
It sounds to be very similar to https://github.com/aws/copilot-cli/blob/77c88c237af66b815648b671332278ba7d5cf120/internal/pkg/cli/svc_exec.go#L291 🤔